addons/nametags/CfgFactionClasses.hpp

```diff
@@ -0,0 +1,23 @@
+class CfgFactionClasses {
+    class OPF_F {
+        GVAR(rankIcons)[] = {QPATHTOF(UI\icons_russia\private_gs.paa), QPATHTOF(UI\icons_russia\corporal_gs.paa), QPATHTOF(UI\icons_russia\sergeant_gs.paa), QPATHTOF(UI\icons_russia\lieutenant_gs.paa), QPATHTOF(UI\icons_russia\captain_gs.paa), QPATHTOF(UI\icons_russia\major_gs.paa), QPATHTOF(UI\icons_russia\colonel_gs.paa)};
+    };
+    class OPF_G_F {
+        GVAR(rankIcons)[] = {QPATHTOF(UI\icons_spain\private_gs.paa), QPATHTOF(UI\icons_spain\corporal_gs.paa), QPATHTOF(UI\icons_spain\sergeant_gs.paa), QPATHTOF(UI\icons_spain\lieutenant_gs.paa), QPATHTOF(UI\icons_spain\captain_gs.paa), QPATHTOF(UI\icons_spain\major_gs.paa), QPATHTOF(UI\icons_spain\colonel_gs.paa)};
+    };
+    class OPF_T_F {
+        GVAR(rankIcons)[] = {QPATHTOF(UI\icons_russia\private_gs.paa), QPATHTOF(UI\icons_russia\corporal_gs.paa), QPATHTOF(UI\icons_russia\sergeant_gs.paa), QPATHTOF(UI\icons_russia\lieutenant_gs.paa), QPATHTOF(UI\icons_russia\captain_gs.paa), QPATHTOF(UI\icons_russia\major_gs.paa), QPATHTOF(UI\icons_russia\colonel_gs.paa)};
+    };
+    class OPF_V_F {
+        GVAR(rankIcons)[] = {QPATHTOF(UI\icons_russia\private_gs.paa), QPATHTOF(UI\icons_russia\corporal_gs.paa), QPATHTOF(UI\icons_russia\sergeant_gs.paa), QPATHTOF(UI\icons_russia\lieutenant_gs.paa), QPATHTOF(UI\icons_russia\captain_gs.paa), QPATHTOF(UI\icons_russia\major_gs.paa), QPATHTOF(UI\icons_russia\colonel_gs.paa)};
+    };
+    class IND_C_F {
+        GVAR(rankIcons)[] = {QPATHTOF(UI\icons_russia\private_gs.paa), QPATHTOF(UI\icons_russia\corporal_gs.paa), QPATHTOF(UI\icons_russia\sergeant_gs.paa), QPATHTOF(UI\icons_russia\lieutenant_gs.paa), QPATHTOF(UI\icons_russia\captain_gs.paa), QPATHTOF(UI\icons_russia\major_gs.paa), QPATHTOF(UI\icons_russia\colonel_gs.paa)};
+    };
+    class IND_F {
+        GVAR(rankIcons)[] = {QPATHTOF(UI\icons_germany\private_gs.paa), QPATHTOF(UI\icons_germany\corporal_gs.paa), QPATHTOF(UI\icons_germany\sergeant_gs.paa), QPATHTOF(UI\icons_germany\lieutenant_gs.paa), QPATHTOF(UI\icons_germany\captain_gs.paa), QPATHTOF(UI\icons_germany\major_gs.paa), QPATHTOF(UI\icons_germany\colonel_gs.paa)};
+    };
+    class IND_G_F {
+        GVAR(rankIcons)[] = {QPATHTOF(UI\icons_spain\private_gs.paa), QPATHTOF(UI\icons_spain\corporal_gs.paa), QPATHTOF(UI\icons_spain\sergeant_gs.paa), QPATHTOF(UI\icons_spain\lieutenant_gs.paa), QPATHTOF(UI\icons_spain\captain_gs.paa), QPATHTOF(UI\icons_spain\major_gs.paa), QPATHTOF(UI\icons_spain\colonel_gs.paa)};
+    };
+};
```

Comment:
```suggestion
}; 

```
Missing newline at end of file.